### PR TITLE
correct the description of Supermarket's search endpoint

### PR DESCRIPTION
**Pull request by robbkidd**
### Description

Corrects the description of Supermarket's search endpoint which looks like it came from Chef Server's API documentation. 😶 

### Definition of Done

Maybe some word smithing? I'm not particularly proud of "Search returns cookbooks that match a query."

### Issues Resolved

@Xorima is trying to build a client library for the API and noticed this documentation was a bit off.

### Check List

- [x] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by mjingle** [comment]
@Xorima @robbkidd Thank you for this PR! @Xorima is correct that documentation reflects the Chef Server's API documentation and is incorrectly applied to the Supermarket API documentation.

The content you corrected is listed under a `search` dtag section. In our set-up, a named dtag section must have the same text throughout the files, especially when re-used in multiple places. Otherwise, the website will not build as expected. Detective work reveals that there are other `search` dtag sections, which apply correctly to the Chef Server's API documentation. Seems like the `search` dtag has evolved and was applied differently over time. 

My current solution to to remove the `search` dtag label from your PR's file. Am currently evaluating if the other 8 files would be eligible for this also. (See file list below, with the line number that the `search` dtag is located.) More updates soon!
*  api_chef_server.rst, ln 4722
*  chef_overview.rst, ln 459
* chef_search.rst, ln 6
* dsl_recipe.rst, ln 696
* knife_search.rst, ln 6
* manage.rst, ln 32
* recipes.rst, ln 597
* server_monitor.rst, ln 443

**Comment by robbkidd** [comment]
Making a note here to figure out how dtags work. 😁

**Comment by mjingle** [comment]
@robbkidd No worries. Hopefully it's not something used for much longer and I'm used to troubleshooting for it. dtags labels are something that we use as a part of Sphinx. Basically, you evaluate whether the content between the beginning and end of a named dtag section match *exactly* to the content of a same-named dtag section in another file. It helps with keeping things consistent, but over time as things evolve, the dtag labels can become erroneous. Often when a description is changed in the docs, other files have to be updated as well.

**Comment by robbkidd** [comment]
@mjingle A-ha. Wellll, certainly the section modified here should _not_ match the search text description for any other thing on the list you provided. So I'm thinking I can remove the search dtag from this file for the edit in this PR, yes?

**Comment by mjingle** [comment]
@robbkidd Correct! I'm looking to see if there might be others incorrectly tagged, but I'll open another PR if that's the case. :)

File: chef_master/source/supermarket_api.rst [hunk 1]
```diff
@@ -416,11 +416,7 @@ The response will return details for a cookbook version, including the license u
 /search
 -----------------------------------------------------
 
-.. tag search
-
-Search indexes allow queries to be made for any type of data that is indexed by the Chef server, including data bags (and data bag items), environments, nodes, and roles. A defined query syntax is used to support search patterns like exact, wildcard, range, and fuzzy. A search is a full-text query that can be done from several locations, including from within a recipe, by using the ``search`` subcommand in knife, the ``search`` method in the Recipe DSL, the search box in the Chef management console, and by using the ``/search`` or ``/search/INDEX`` endpoints in the Chef server API. The search engine is based on Apache Solr and is run from the Chef server.
-
-.. end_tag
+Search returns cookbooks that match a query. The query syntax is a simple, fuzzy text search on cookbook names, cookbook descriptions, and the cookbook owners' usernames.
```
Comment (mjingle) [inline chef_master/source/supermarket_api.rst]: @robbkidd How's this for word-smithing? 
```suggestion
Search performs a fuzzy, keyword search on cookbook names, cookbook descriptions, and the cookbook owners' usernames.
```

Comment (robbkidd) [reply chef_master/source/supermarket_api.rst]: Those are some well-smithed words. Smelted down and hammered into brevity. 👍

**Review by mjingle — APPROVED** [review]
tenor-79815668

**Pull request merged**